test/routing/ArrivalsRouterSpec.scala

```diff
@@ -84,27 +85,83 @@ class ArrivalsRouterSpec extends AnyFreeSpec with Matchers with OptionValues wit
           )
 
           s"when the accept header equals ${acceptHeaderValue.getOrElse("nothing")}, it returns status code $expectedStatus" in {
+            isVersion match {
+              case true  => when(mockAppConfig.enablePhase5).thenReturn(true)
+              case false => when(mockAppConfig.enablePhase5).thenReturn(false)
+            }
 
             val request =
               FakeRequest(method = callValue.method, uri = callValue.url, body = <test></test>, headers = arrivalsHeaders)
             val result = call(sutValue, request)
 
             status(result) mustBe expectedStatus
-            contentAsJson(result) mustBe Json.obj("version" -> 2) // ensure we get the unique value to verify we called the fake action
+
+            isVersion match {
+              case true => contentAsJson(result) mustBe Json.obj("version" -> 2) // ensure we get the unique value to verify we called the fake action
+              case false =>
+                contentAsJson(result) mustBe Json.obj(
+                  "message" -> "CTC Traders API version 2 is not yet available. Please continue to use version 1 to submit transit messages.",
+                  "code"    -> "NOT_ACCEPTABLE"
+                )
+            }
```

Comment:
Can we just use an `if` statement here? We really shouldn't use pattern matching on simple booleans...